src/App.js

```diff
@@ -46,11 +48,18 @@ const routes = [
     createRoute("/organization/offers", withUserAuth(OffersView, ["organization"], {getOffers: () => offersAPI.getOffersFromOrganization(), buttons: hrOfferButtons})),
     createRoute("/login", <LoginForm />),
     createRoute("/register", <RegistrationRouting />),
-    createRoute("*", <div>Page</div>)
+    createRoute("/settings", <SettingsOverlay />, {marginTop: "1em"}),
+    // TODO: Does not work, dont know why
+    createRoute("/settings/profile", <Redirect to={{path: "/settings", state: { subPage: "profile" }}} />),
+    createRoute("/settings/settings", <Redirect to={{path: "/settings", state: { subPage: "settings" }}} />),
+    createRoute("/settings/files", <Redirect to={{path: "/settings", state: { subPage: "files" }}} />),
+    createRoute("*", <Redirect to="/" />)
```

Comment:
Po to jest strona 404, żeby czegoś takiego nie było. User wchodząc na jakąś stronę, jeśli będzie wywalony na stronę główną będzie na maxa confused